buildspec.yml

```diff
@@ -15,7 +17,7 @@ phases:
       - echo Building the Docker image...
       - chmod +x ./update_data.sh
       - echo Getting latest GeoLite2 ASN DB
-      - wget https://geolite.maxmind.com/download/geoip/database/GeoLite2-ASN.tar.gz
+      - wget https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-ASN&license_key=`$SUA_GEOLITE_ACCESS_KEY`&suffix=tar.gz
```

Comment:
`wget` treats query parameters as unique filenames, so the file downloaded with this command would be named something like `geoip_download?edition_id=GeoLite2-ASN&license_key=[OUR_LICENSE_KEY]&suffix=tar.gz` rather than the desired `GeoLite2-ASN.tar.gz`. Our key isn't currently working, so I couldn't verify this with the above URL specifically, but I did verify that behavior when I downloaded a page from my own site with arbitrary query parameters.

We can specify an output filename using the `-O` / `--output` flag. Alternatively, we could use `curl`, which downloads single files (my understanding is that `wget` is primarily built for downloading a whole site by following links). We could also consider adding flags to enable verbose output, which could help in future debugging.